[BUG] Could not compile for Android(Termux)

<!-- If you're here to report about a "No asset found" error, please make sure that an hour has been passed since the last release was made. -->

## What did you expect to happen?

Should compile without errors.

## What actually happened?
```
error[E0425]: cannot find function, tuple struct or tuple variant `SkipStep` in this scope
   --> src/steps/os/unix.rs:160:20
    |
160 |         return Err(SkipStep(String::from("Gnome shell extensions are unregistered in DBus")).into());
    |                    ^^^^^^^^ not found in this scope
    |
help: consider importing this tuple struct
    |
3   | use crate::error::SkipStep;
    |
```


## Additional details
- Which operating system or Linux distribution are you using?

Cross-compiling on ubuntu 20.04

- How did you install Topgrade?
*Not applicable*
<!--
Run `topgrade --dry-run` to see which commands Topgrade is running.
If the command seems wrong and you know why pleaes tell us so.
If the command seems fine try to run it yourself
and tell us if you got a different result from Topgrade.
-->

## Suggested solution
This patch makes it work, but I thought this should be reported since you may have to patch at other places too.
```patch
--- a/src/steps/os/unix.rs	2022-04-12 00:55:54.772532982 +0530
+++ b/src/steps/os/unix.rs	2022-04-12 00:59:10.312532907 +0530
@@ -1,4 +1,4 @@
-#[cfg(target_os = "linux")]
+#[cfg(any(target_os = "linux", target_os = "android"))]
 use crate::error::SkipStep;
 use crate::error::TopgradeError;
 use crate::execution_context::ExecutionContext;
```

<details><pre>
Paste the output of `topgrade -v`
*Not applicable*
</pre></details>

--- a/src/main.rs
+++ b/src/main.rs
@@ -280,7 +280,7 @@ fn run() -> Result<()> {
         runner.execute(Step::Tmux, "tmux", || tmux::run_tpm(&base_dirs, run_type))?;
         runner.execute(Step::Tldr, "TLDR", || unix::run_tldr(run_type))?;
         runner.execute(Step::Pearl, "pearl", || unix::run_pearl(run_type))?;
-        #[cfg(not(target_os = "macos"))]
+        #[cfg(not(any(target_os = "macos", target_os = "android")))]
         runner.execute(Step::GnomeShellExtensions, "Gnome Shell Extensions", || {
             unix::upgrade_gnome_extensions(&ctx)
         })?;

--- a/src/steps/generic.rs
+++ b/src/steps/generic.rs
@@ -285,9 +285,9 @@ pub fn run_stack_update(run_type: RunType) -> Result<()> {
 
 pub fn run_tlmgr_update(ctx: &ExecutionContext) -> Result<()> {
     cfg_if::cfg_if! {
-        if #[cfg(target_os = "linux")] {
+        if #[cfg(any(target_os = "linux", target_os = "android"))] {
             if !ctx.config().enable_tlmgr_linux() {
-                return Err(SkipStep(String::from("tlmgr must be explicity enabled in the configuration to run in Linux")).into());
+                return Err(SkipStep(String::from("tlmgr must be explicity enabled in the configuration to run in Android/Linux")).into());
             }
         }
     }

--- a/src/steps/os/unix.rs
+++ b/src/steps/os/unix.rs
@@ -135,7 +135,7 @@ pub fn run_fish_plug(ctx: &ExecutionContext) -> Result<()> {
     ctx.run_type().execute(&fish).args(&["-c", "plug update"]).check_run()
 }
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(not(any(target_os = "android", target_os = "macos")))]
 pub fn upgrade_gnome_extensions(ctx: &ExecutionContext) -> Result<()> {
     let gdbus = require("gdbus")?;
     require_option(

